src/pages/home/_trade-types-mobile.js

```diff
@@ -74,6 +84,23 @@ const TradeTypeSlide = ({ icon, title, description }) => {
                     <Text size="var(--text-size-sm)" align="center">
                         {description}
                     </Text>
+                    <StyledLinkText
+                        weight="bold"
+                        color="red"
+                        mt="16px"
+                        size="16px"
+                        ariaLabel={localize('Margin')}
+                        to={
+                            icon == <MarginLogo />
```

Comment:
on line `116` to `138` we have definition for each trade types, please add redirection link there, so it can be used here, instead of adding conditionals.